### ISSUE TITLE
feat: support verbose log in dev mode?

### ISSUE THREAD

**Issue by imcuttle** [jameslnewell/xhr-mock#110]
Now xhr-mock is silent when mocking，It's confused when debug

**Comment by mkay581** [comment]
Agree. @imcuttle did you ever find a workaround? Hard to figure out why a component isn't rendering.
